test/run

```diff
@@ -44,7 +44,13 @@ main() {
         write out
         quit!
       "
-      ${test}/../src/kak out -n -u -e "$kak_commands"
+      if [ -z ${TEST_LOAD_KAKRC+x} ]; then
```

Comment:
>${TEST_LOAD_KAKRC+x}

Is the `+x` inside the braces a syntax I don't know about, or a typo ?

I'm not too sure about the way the option is detected too, using an environment variable as a flag seems wonky, especially when you have a makefile available.